### PR TITLE
{bio} [GCC/8.2.0-2.31.1] bpp-core/bpp-seq/bpp-phyl 2.4.1

### PR DESCRIPTION
**Pull request by omula**

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@omula did you consider using `GCCcore` instead of `GCC`? Although you'll need to add `binutils` as a `builddependency`, the advantage is that `GCCcore` is a subtoolchain of `foss` and `intel`, which would make these potentialy more useful in other situations (?)

**Comment by omula** [comment]
@migueldiascosta thanks for having a look into it.
Shouldn't I use GCCcore always instead of GCC? Do you have any suggestion or rules for choosing toolchain?

**Comment by migueldiascosta** [comment]
@omula the difference is only relevant when used as a dependency and/or with hierarchical module naming schemes, but looking at the easyconfigs for the latest toolchains I'd say that `GCCcore` is becoming the default choice over `GCC`

On the other hand, there's this: https://github.com/easybuilders/easybuild-easyconfigs/issues/6366. @boegel?

In any case, it's not a blocker for this PR

**Comment by boegel** [comment]
We usually use `GCCcore` for stuff where performance doesn't really matter, while we use `GCC` (+ `iccifort`) when it does matter.

I'd say this falls in the latter category, so good as is...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2632.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8764ae4cc24701a00957b210b760bfd6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/cbc96baccb819951f9e9bca19b5402bc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2068.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e9c8cabbfd8e51e26d13c54518eb3c0c for a full test report.

File: easybuild/easyconfigs/b/bpp-core/bpp-core-2.4.1-GCC-8.2.0-2.31.1.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+github_account = 'BioPP'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['1150b8ced22cff23dd4770d7c23fad11239070b44007740e77407f0d746c0af6']
+
+builddependencies = [('CMake', '3.13.3')]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["lib64"]
```
Comment (boegel) [inline easybuild/easyconfigs/b/bpp-core/bpp-core-2.4.1-GCC-8.2.0-2.31.1.eb]: @omula I would make this a bit more specific:

```python
sanity_check_paths = {
    'files': ['lib64/libbpp-core.a', 'lib64/libbpp-core.%s' % SHLIB_EXT],
    'dirs': ['include/Bpp']
}

File: easybuild/easyconfigs/b/bpp-phyl/bpp-phyl-2.4.1-GCC-8.2.0-2.31.1.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    ('bpp-core', '2.4.1'),
+    ('bpp-seq', '2.4.1'),
+]
+
+configopts = '-Dbpp-core_DIR=${EBROOTBPPMINCORE}/lib64/cmake/bpp-core/ '
+configopts += '-Dbpp-seq_DIR=${EBROOTBPPMINSEQ}/lib64/cmake/bpp-seq/ '
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["lib64"]
```
Comment (boegel) [inline easybuild/easyconfigs/b/bpp-phyl/bpp-phyl-2.4.1-GCC-8.2.0-2.31.1.eb]: @omula I would make this a bit more specific:

```python
sanity_check_paths = {
    'files': ['lib64/libbpp-phyl.a', 'lib64/libbpp-phyl.%s' % SHLIB_EXT],
    'dirs': ['include/Bpp']
}

File: easybuild/easyconfigs/b/bpp-seq/bpp-seq-2.4.1-GCC-8.2.0-2.31.1.eb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+checksums = ['dbfcb04803e4b7f08f9f159da8a947c91906c3ca8b20683ac193f6dc524d4655']
+
+builddependencies = [('CMake', '3.13.3')]
+
+dependencies = [('bpp-core', '2.4.1')]
+configopts = ' -Dbpp-core_DIR=${EBROOTBPPMINCORE}/lib64/cmake/bpp-core/ '
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["lib64"]
```
Comment (boegel) [inline easybuild/easyconfigs/b/bpp-seq/bpp-seq-2.4.1-GCC-8.2.0-2.31.1.eb]: @omula I would make this a bit more specific:

```python
sanity_check_paths = {
    'files': ['lib64/libbpp-seq.a', 'lib64/libbpp-seq.%s' % SHLIB_EXT],
    'dirs': ['include/Bpp']
}

File: easybuild/easyconfigs/b/bpp-core/bpp-core-2.4.1-GCC-8.2.0-2.31.1.eb (hunk 1 above)
Comment (omula) [reply easybuild/easyconfigs/b/bpp-core/bpp-core-2.4.1-GCC-8.2.0-2.31.1.eb]: done

File: easybuild/easyconfigs/b/bpp-phyl/bpp-phyl-2.4.1-GCC-8.2.0-2.31.1.eb (hunk 2 above)
Comment (omula) [reply easybuild/easyconfigs/b/bpp-phyl/bpp-phyl-2.4.1-GCC-8.2.0-2.31.1.eb]: done

File: easybuild/easyconfigs/b/bpp-seq/bpp-seq-2.4.1-GCC-8.2.0-2.31.1.eb (hunk 3 above)
Comment (omula) [reply easybuild/easyconfigs/b/bpp-seq/bpp-seq-2.4.1-GCC-8.2.0-2.31.1.eb]: done

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/66c4c33be5995419c2c596e59efe1239 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2696.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2d00037a38a7941fc330f2a72d171791 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/79d0505aabb11c024ea5838ef8b0ad53 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @omula!

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node301.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/fc1d4b41d064e85596808f3a146d848a for a full test report.

**Comment by mboisson** [comment]
I would say that this PR should be superseded by https://github.com/easybuilders/easybuild-easyconfigs/pull/11113 in the future. Those tools are actually a bundle which should probably be installed together.

**Pull request merged**